frontend/src/pages/Policy/policyContent/HowToUseTheDataContent.tsx

```diff
@@ -0,0 +1,209 @@
+				},
+				{
+					step: 'Understand Impact',
+					description: 'Recognize how these gaps can affect policy-making.'
+				},
+				{
+					step: 'Advocate',
+					description: 'Demand better data collection in these areas to ensure informed decision-making.'
+				}
+			]
+		},
+		customCard: <CustomUnknownMap/>,
+	},
+	{
+		title: 'Relative Inequity Chart',
+		description: 'Shows how disparities in gun violence exposure have evolved among different demographic groups.',
+        sectionId: '#relative-inequity-chart',
+		details: {
+			demographicGranularities: ['Race/ethnicity', 'sex', 'age'],
+			geographicLevels: ['National', 'states', 'counties', 'territories'],
+			alternateBreakdowns: ['Table view'],
+			howToUse: [
+				{
+					step: 'Choose Demographics',
+					description: 'Select which disparities to visualize, such as by race or age.'
+				},
+				{
+					step: 'Track Changes',
+					description: 'Identify if these inequities are worsening or improving.'
+				},
+				{
+					step: 'Advocate',
+					description: 'Use these trends to lobby for policies that address growing inequalities.'
+				}
+			]
+		},
+		customCard: <CustomShareTrendsLineChart
+		fips={new Fips('00')}
+		dataTypeConfig={METRIC_CONFIG['gun_deaths_black_men'][0]}
+		demographicType='age'
+		reportTitle='Historical relative inequity of gun homicides among Black (NH) men in the United States'
+		isCompareCard={false} 
+	  />,
+	},
+	{
+		title: 'Population vs. Distribution Stacked Bar Chart',
+		description: 'Illustrates how the prevalence of gun violence in certain demographics compares to their population size.',
+        sectionId: '#population-vs-distribution-stacked-bar-chart',
+		details: {
+			demographicGranularities: ['Race/ethnicity', 'sex', 'age'],
+			geographicLevels: ['National', 'states', 'counties', 'territories'],
+			alternateBreakdowns: 'N/A',
+			howToUse: [
+				{
+					step: 'View Data',
+					description: 'Look at the proportion of each demographic in the overall population versus their share of gun violence victims.'
+				},
+				{
+					step: 'Highlight Disproportionality',
+					description: 'Point out any significant disparities.'
+				},
+				{
+					step: 'Advocate',
+					description: 'Argue for interventions that help disproportionately affected groups.'
+				}
+			]
+		},
+		customCard: <CustomDisparityBarChart
+		fips={new Fips('00')}  
+		dataTypeConfig={METRIC_CONFIG['gun_deaths_black_men'][0]} 
+		demographicType='urbanicity'
+		reportTitle='Population vs. distribution of total gun homicides among Black (NH) men in the United States'
+	  />,
+	},
+	{
+		title: 'Breakdown Summary Data Table',
+		description: `'Rates' show how often something happens within a specific group per 100,000 people. 'Share' tells you what percentage of all cases come from a particular group, while 'Population share' shows the percentage of the total population that the group makes up.`,
```

Comment:
we also present rates as pct_rate sometimes so should adjust this: 'Rates' show how often something happens within a specific group, either per 100,000 people or per 100 people (as a percent).